fastly_nsq.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_development_dependency 'awesome_print', '~> 1.6'
-  gem.add_development_dependency 'bundler', '~> 1.11.2'
+  gem.add_development_dependency 'bundler', '~> 1.12'
```

Comment:
The best way is `'~> 1.12'` as written since it allows upgrades to compatible future versions.  If we need to be at a particular bug fix level, `'~> 1.12', '>= 1.12.2'` would allow us to use newer compatible versions.